src/opencl_fvde_fmt_plug.c

```diff
@@ -0,0 +1,355 @@
+/*
+ * This software is Copyright (c) 2017 Dhiru Kholia <kholia at kth.se> and
+ * Copyright (c) 2013 Lukas Odzioba <ukasz at openwall dot net> and it is
+ * hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ */
+#ifdef HAVE_OPENCL
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_opencl_fvde;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_opencl_fvde);
+#else
+
+#include <string.h>
+
+#include "misc.h"
+#include "arch.h"
+#include "fvde_common.h"
+#include "common.h"
+#include "stdint.h"
+#include "formats.h"
+#include "options.h"
+#include "common-opencl.h"
+
+#define FORMAT_LABEL            "FVDE-OpenCL"
```

Comment:
You should change this to "FVDE-opencl". At least, the format 'filter' is not working properly.